src/OSPSuite.Core/Domain/Descriptors/MatchTagCondition.cs

```diff
@@ -47,16 +46,9 @@ public override bool Equals(object otherObject)
       public bool Equals(MatchTagCondition other)
       {
          if (other == null) return false;
-         return string.Equals(Tag,other.Tag);
+         return string.Equals(Tag, other.Tag);
       }
 
-      public override int GetHashCode()
-      {
-         //Match and NoMatch CANNOT return the same hash for the same tag
-         int hash = 29; //one prime number that should differ from the one in Not Match
```

Comment:
THis was a bit cumbersome all of that. My goal years ago was to ensure that MatchTag "XX" and NotMatchTag "XX" where not seen as equal
Since they have a different display, we should  use that instead